src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -381,7 +381,7 @@ private function convertType(Type $type, bool $input = false, string $mutationNa
                     return null;
                 }
 
-                $graphqlType = $this->getResourceObjectType($resourceClass, $resourceMetadata, $input, $mutationName, $depth);
+                $graphqlType = $this->getResourceObjectType($resourceMetadata->getAttribute($input ? 'input_class' : 'output_class', $resourceClass), $resourceMetadata, $input, $mutationName, $depth);
```

Comment:
```suggestion
                $graphqlType = $this->getResourceObjectType(false !== $attribute = $resourceMetadata->getAttribute($input ? 'input_class' : 'output_class', $resourceClass) ? $attribute : $resourceClass, $resourceMetadata, $input, $mutationName, $depth);
```


because `input_class` or `output_class` can be set to `false`, but the method expects `string` as first parameter ?